DOCS.md

```diff
@@ -231,15 +231,16 @@ The following parameters are used to configure the image:
 | `registry`             | name of the registry for the repository                            | `true`   | `index.docker.io` | `PARAMETER_REGISTRY`<br>`KANIKO_REGISTRY`                         |
 | `repo`                 | name of the repository for the image                               | `true`   | `N/A`             | `PARAMETER_REPO`<br>`KANIKO_REPO`                                 |
 | `sha`                  | SHA-1 hash generated for commit                                    | `true`   | **set by Vela**   | `PARAMETER_SHA`<br>`KANIKO_SHA`<br>`VELA_BUILD_COMMIT`            |
-| `snapshot_mode`        | control how to snapshot the filesystem. - options: `full`, `redo`, or `time` | `false`  | `N/A`             | `PARAMETER_SNAPSHOT_MODE`<br>`KANIKO_SNAPSHOT_MODE`               |
+| `use_new_run`          | use experimental run implementation for detecting changes without requiring file system snapshots | `false` | `N/A` | `PARAMETER_USE_NEW_RUN`<br>`KANIKO_USE_NEW_RUN` |
```

Comment:
```suggestion
| `use_new_run`          | use experimental run implementation for detecting changes without requiring file system snapshots | `false` | `false` | `PARAMETER_USE_NEW_RUN`<br>`KANIKO_USE_NEW_RUN` |
```

i think the default is `false`. it would match how we treat other boolean values.